config/cp4waiops-custom/templates/x-small-idle-workload.yaml

```diff
@@ -0,0 +1,509 @@
+{{- if eq .Values.profile "x-small-idle"  }}
```

Comment:
Can the name of this file be consistent w/ the profile value? i.e. file name to be `x-small-idle.yaml` or profile value to be `x-small-idle-workload`, so as to avoid the possible typo when use it.

Reply:
Agree. The file name will be updated as `x-small-idle.yaml` to match profile `x-small-idle` .